### DISCUSSION TITLE
I'm new to SP API and still haven't received any response.

### DISCUSSION THREAD

**Discussion by cnr91** [amzn/selling-partner-api-models#2827] [General]
I'm trying to take orders to understand the situation. But amazon does not allow access.

```
import requests
from sp_api.api import Orders
from sp_api.base import Marketplaces

from keys import CLIENT_CONFIG

if __name__ == '__main__':
    today = date.today()
    d1 = today.strftime("%Y-%m-%d")

    res = Orders(credentials=CLIENT_CONFIG, marketplace=Marketplaces.AU)
    data = res.get_orders(CreatedAfter="2022-01-30", CreatedBefore=d1)
```

I am getting such an error. 
```
    raise exception(error, headers=res.headers)
sp_api.base.exceptions.SellingApiForbiddenException: [{'message': 'Access to requested resource is denied.', 'code': 'Unauthorized', 'details': 
''}]
```

**Comment by stefnats** [comment]
Hey there @cnr91 

could you throw some light in here? what library are you using and how are you initializing things. You should at least init somehow your access/secret keys, app id and for the seller you are requesting things for the refresh token.

I think you are using `saleweaver/python-amazon-sp-api` (Python library), is that right?

I admit, i'm not firm with python, maybe you could also try to ask there.
